int32 SUCCESS          = 0
int32 OBSTACLE_LASER   = 1
int32 OBSTACLE_VOLATILE= 2
int32 OBSTACLE_TAG     = 3
int32 OBSTACLE_ROVER   = 4
int32 PATH_FAIL        = 5
int32 USER_ABORT       = 6
int32 TIMEOUT          = 7
int32 OBSTACLE_CORNER  = 8

int32 result 
int32 obstacle
int8 obstacle_data
float32 distance
